lib/core/validation/validators/validator-enumeration.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+(function(root) {
+
+  var _ = require('lodash');
```

Comment:
in `v1` verson of this lib we don't `require` things.  in `v2` we leverage Webpack and can `require` modules